packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -220,10 +224,31 @@ const View = BaseView.extend({
       if (ChildView === CommunicationPreferencesView) {
         return areCommunicationPrefsVisible;
       }
+      if (ChildView === SubscriptionView) {
+        return this._isSubscriptionsManagementVisible();
+      }
       return true;
     });
   },
 
+  /**
+   * Should the subscriptions management panel be displayed?
+   *
+   * @returns {Boolean}
+   * @private
+   */
+  _isSubscriptionsManagementVisible () {
+    if (! this._subscriptionsManagementEnabled) {
+      return false;
+    }
+    const acceptedLanguages = (this.navigator.languages || [])
```

Comment:
The locale the rest of the FxA UI is being displayed in is defined in [config.lang](https://github.com/mozilla/fxa/blob/68978f34c9ed96536a6adc490f6d2f6a178d34b6/packages/fxa-content-server/app/scripts/lib/config-loader.js#L33), where [config is options.config in the initialize function](https://github.com/mozilla/fxa/pull/940/files#diff-33ca28c5444ea7de91653e837ef072caR70).

Is it true the only gate is locale, or is it the user's location, i.e., the country in which they are located? If it's the country, we don't have that info on the front end currently, but should be able to add it with minimal fuss, perhaps passed in with [serializedConfig in get-index](https://github.com/mozilla/fxa/pull/940/files#diff-da283abf354df26a8bebd2d236b18f39R55).

Reply:
I think so far, it's only locale because the payment pages won't be localized at launch. But, I think we're going to have a follow-up to this that will also hide the button if the user doesn't have any active subscriptions - which, itself, will be gated by geographic market.

Reply:
Hmm, looking at config.lang, that's just language and not locale. That's probably good enough here for now, because along with geographic market I think we're not applying l10n to the payment pages for MVP